src/app/onboarding/components/archive-creation-start-screen/archive-creation-start-screen.component.scss

```
@@ -31,9 +31,16 @@ p {
   @media screen and (max-width: 600px) {
     width: 100%;
   }
-  & > p {
-    font-size: 50px;
-    font-family: 'UsualRegular', sans-serif;
+  & > .greeting {
+    font-size: 4rem;
+    font-family: 'UsualLight', sans-serif;
+    line-height: 4rem;
```

```suggestion
    font-size: 3.5rem;
    font-family: 'UsualLight', sans-serif;
    line-height: 4.5rem;
```
If we were converting the units from the Figma design to rem it would be these values, right? (I'm pretty sure 1rem = 16px)